x/hub-genesis/keeper/keeper.go

```diff
@@ -56,21 +56,16 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 
 // lock coins by sending them to an escrow address
 func (k Keeper) lockRollappGenesisTokens(ctx sdk.Context, sourceChannel string) error {
-	account := k.accountKeeper.GetModuleAccount(ctx, types.ModuleName)
-
 	// get spendable coins in the module account
+	account := k.accountKeeper.GetModuleAccount(ctx, types.ModuleName)
 	spendable := k.bankKeeper.SpendableCoins(ctx, account.GetAddress())
 
-	// send coins to the escrow address
-	if spendable.Empty() {
-		return types.ErrGenesisNoCoinsOnModuleAcc
+	// validate it's enough for the required tokens
+	locked := k.GetLocked(ctx)
```

Comment:
we already retrieve `locked` above in the `TriggerGenesisEvent`, so consider just passing the tokens down instead of retrieving `locked` again